container.go

```diff
@@ -517,7 +529,7 @@ func (c *Container) start() error {
 	}
 
 	// Attach devices
-	if err := c.attachDevices(); err != nil {
+	if err := c.attachDevices(false); err != nil {
```

Comment:
all devices including vfio will be hot plugged?

Reply:
Yes, although we havent yet added support for vfio hotplug.

Reply:
Have you tried hot-plugging the vfio device in a pci bridge?